### PR TITLE
feature(wallet-mobile): add theme support for settings & collateral funnels

### PR DESCRIPTION
**Pull request by SorinC6**
task: https://emurgo.atlassian.net/browse/YOMO-1177

### CODE REVIEW

File: apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx [hunk 1]
```diff
@@ -115,27 +119,33 @@ const messages = defineMessages({
...
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      backgroundColor: color.gray.min,
+    },
+    contentContainer: {
+      padding: 16,
+    },
+    heading: {
+      fontSize: 16,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx]: can it consume any typography?

File: apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx [hunk 2]
```diff
@@ -115,27 +119,33 @@ const messages = defineMessages({
...
+      lineHeight: 24,
+      paddingBottom: 20,
+    },
+    warning: {
+      color: color.magenta[500],
+      fontSize: 14,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx]: :arrow_up:

File: apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx (hunk 1 above)
Comment (SorinC6) [reply apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx]: yes!l let me check all the `fontSize` again

**Review by stackchain — APPROVED** [review]

**Comment by jorbuedo** [comment]
Every time I see the `useStyles` functions I worry a bit about react performance.

Inside, useTheme is a useContext, so it should only trigger renders when the provider actually changes the objects inside (theme, colorScheme, etc.). Which is fine.

But other causes of rerenders in the parent component will basically run useStyles again, which calls StyleSheet.create with a possibly big object full of spreads inside. 

I'm not sure if the cost of all these calls vs the cost of adding memos everywhere would be worth it though.

**Review by jorbuedo — APPROVED** [review]

**Pull request merged**